src/string.rs

```diff
@@ -431,7 +419,7 @@ impl String {
   ) -> OneByteConst {
     is_ascii(buffer);
     OneByteConst {
-      vtable: &ONE_BYTE_CONST_VTABLE.2,
+      vtable: &ONE_BYTE_CONST_VTABLE.delete1,
```

Comment:
This is interesting -- you can take an address of a const?

Reply:
> This is interesting -- you can take an address of a const?

Seems so. It was originally a static but clippy told me to make it a const.

Reply:
![image](https://github.com/denoland/rusty_v8/assets/19621382/296b7abd-aca4-4e66-b36b-30c3a66ea4a0)